README.md

```diff
@@ -42,7 +42,7 @@ Let us know about any issues with this document.
 
 ## License
 
-Copyright 2019 Autodesk
+Copyright © 2020 Autodesk
```

Comment:
Let's go ahead and change it to 2021

Reply:
Whoops. That was definitely a result of me forgetting it is 2021. 🤣